library/build.gradle

```diff
@@ -21,6 +21,17 @@ android {
     defaultConfig {
         minSdkVersion 14
         targetSdkVersion 22
+
+        def projects = rootProject.getProject().getChildProjects()
+        projects.each {
+            def project = it.value
+            if (project.hasProperty('android')) {
```

Comment:
@devisnik had to do it this way to avoid java modules

no need for the `.?` dereferencing below, we'd want it to crash if `downloadAuthority` is not set